spec/app_spec.rb

```diff
@@ -0,0 +1,34 @@
+ENV['APP_ENV'] = 'test'
+
+require_relative 'spec_helper'
+require_relative '../app'
+require 'rack/test'
+
+describe 'Server Service' do
+  include Rack::Test::Methods
+
+  def app
+    Sinatra::Application
+  end
+
+  it "should load the home page" do
```

Comment:
Lembre-se que por padrão no rspec não documentamos testes com should. As mensagens devem ser assertivas.

Reply:
corrigido